Source/WebCore/page/Quirks.h

```diff
@@ -70,7 +70,6 @@ class Quirks {
     bool shouldDispatchSimulatedMouseEvents(const EventTarget*) const;
     bool shouldDispatchedSimulatedMouseEventsAssumeDefaultPrevented(EventTarget*) const;
     std::optional<Event::IsCancelable> simulatedMouseEventTypeForTarget(EventTarget*) const;
-    bool shouldMakeTouchEventNonCancelableForTarget(EventTarget*) const;
```

Comment:
Where is the part of the patch that removes the caller to this function? If there is no caller then this doesn’t remove a quirk, it just removes dead code.

Reply:
Or maybe the caller is in internal Apple touch code? If so, that removal needs to happen first.

Reply:
Make sense! Let me explore.

Reply:
This was added in 2019
https://github.com/WebKit/WebKit/commit/7dfb1414d17f5a78f1c4ce0b5126580f900c478b

but I wonder why now… 
It was never called in WebKit.

There's a mention of an internal call in this comment.
https://bugs.webkit.org/show_bug.cgi?id=200573#c9